### PR TITLE
Feature/MeshStructured

### PR DESCRIPTION
**Pull request by LANTZT**
- Add possibility to build "deformed" mesh from coordinates files
- Add polygon selection of "bad" areas we don't want to take in account

### CODE REVIEW

File: feel/feeldiscr/meshstructured.cpp [hunk 1]
```diff
@@ -22,11 +22,17 @@
  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 #include <feel/feeldiscr/meshstructured.hpp>
+#include <boost/geometry.hpp>
```
Comment (prudhomm) [inline feel/feeldiscr/meshstructured.cpp]: Is there a requirement on Boost and Boost.geometry version ?

Comment (prudhomm) [inline feel/feeldiscr/meshstructured.cpp]: We should probably add a note on the use of Boost Geometry in Feel++ documentation. this is new definitely

File: feel/feeldiscr/meshstructured.cpp [hunk 2]
```diff
@@ -72,6 +78,40 @@ MeshStructured::MeshStructured( int nx, int ny, double pixelsize, holo3_image<fl
...
     node_type coords( 2 );
 
+    
+    // polygon build
+    std::vector<polygon_type> list_poly ;
+    //std::cout << list_poly.size() << std::endl;
+    polygon_type poly;
+    bool inPoly = false;
+    bool testInPoly = false;
+    if (withPoly)
```
Comment (prudhomm) [inline feel/feeldiscr/meshstructured.cpp]: @LANTZT is this (polygonal extraction) described in the documentation of the holo3 project ?

Comment (prudhomm) [inline feel/feeldiscr/meshstructured.cpp]: Moreover is polygonal mesh extraction tested in the testsuite ?

Comment (LANTZT) [inline feel/feeldiscr/meshstructured.cpp]: Yes at the first question 
And No to the second

**Pull request merged**